### PR TITLE
refine containerd nri plugin optimizer docs

### PR DESCRIPTION
**Pull request by changweige**
Add a brief introduction section to README and refine the optimizer docs a bit

### CODE REVIEW

File: docs/opmitize_nydus_image.md [hunk 1]
```diff
@@ -66,14 +68,14 @@ EOF
 
 Restart the containerd service.
 
-```
+```condole
```
Comment (sctb512) [inline docs/opmitize_nydus_image.md:71]: console

File: docs/opmitize_nydus_image.md [hunk 2]
```diff
@@ -1,34 +1,36 @@
-# Optimizing the nydus image
+# Optimize a nydus image
+
+To improve the prefetch hit rate, we can specify a prefetch table when converting an OCI image to a nydus image. The prefetch table is workload related and we should generate the list of accessed files in order when the workload is running. Nydus-snapshotter has implemented an optimizer with Containerd NRI plugin to optimize the nydus image. The optimizer is image format independent and requires NRI 2.0, which is available in containerd 1.7.0 (not yet released). The optimizer subscribe container and sandbox events to watch what files are opened and read, etc. during container application is starting up. This enables nydus image
```
Comment (sctb512) [inline docs/opmitize_nydus_image.md:3]: Now the optimizer only subscribes to `StartContainer` event without sandbox events.

**Review by sctb512 — COMMENTED** [review]
Others, LGTM

File: docs/opmitize_nydus_image.md (hunk 2 above)
Comment (changweige) [reply docs/opmitize_nydus_image.md]: fixed

**Review by sctb512 — APPROVED** [review]

**Pull request merged**